src/contracts/acrossConfigStore.ts

```diff
@@ -14,21 +14,21 @@ const L1TokenConfigSs = object({
   rateModel: RateModelSs,
   routeRateModel: optional(record(string(), RateModelSs)),
 });
-export type RateModel = Infer<typeof RateModelSs>;
-export type L1TokenConfig = Infer<typeof L1TokenConfigSs>;
+type RateModelSS = Infer<typeof RateModelSs>;
```

Comment:
Renamed these types and removed the `export` so that other classes don't confuse this type with the `RateModel` type in `lpFeeCalculator/rateModel.ts`